mm2src/coins/lp_coins.rs

```diff
@@ -246,9 +246,9 @@ use hd_wallet::{HDAddress, HDAddressId};
 use qrc20::Qrc20ActivationParams;
 use qrc20::{qrc20_coin_from_conf_and_params, Qrc20Coin, Qrc20FeeDetails};
 use qtum::{Qrc20AddressError, ScriptHashTypeNotSupported};
-use rpc_command::init_create_account::{CreateAccountTaskManager, CreateAccountTaskManagerShared};
-use rpc_command::init_scan_for_new_addresses::{ScanAddressesTaskManager, ScanAddressesTaskManagerShared};
-use rpc_command::init_withdraw::{WithdrawTaskManager, WithdrawTaskManagerShared};
+use rpc_command::{init_create_account::{CreateAccountTaskManager, CreateAccountTaskManagerShared},
+                  init_scan_for_new_addresses::{ScanAddressesTaskManager, ScanAddressesTaskManagerShared},
+                  init_withdraw::{WithdrawTaskManager, WithdrawTaskManagerShared}};
```

Comment:
Why do you prefer this approach?
Personally, I see several downsides to this:
* If you move `init_create_account` module from `rpc_command` to another place, you will need to cut the `init_create_account::{CreateAccountTaskManager, CreateAccountTaskManagerShared}` line, and put it alongside.
* If you removed `init_create_account` functionality, you need to select `init_create_account::{CreateAccountTaskManager, CreateAccountTaskManagerShared}` and then remove it.
* If you want to import another module from `rpc_command`, you need to find the right closing bracket between `WithdrawTaskManagerShared ` and `;`, then **add comma** and then change.

Reply:
You mean why I import `init_create_account`, `init_scan_for_new_addresses` and `init_withdraw` under `rpc_command`? If so, it looks more decent than using multiple `use rpc_command` to me and it's default behaviour of `rust_analyzer`.